### PR TITLE
Removes the clown

### PR DESCRIPTION
**Pull request by harmonyn**
...mask typo that always bugged me.

### CODE REVIEW

**Comment by Xhuis** [comment]
# ___AYYYYYYYY___

**Comment by AnturK** [comment]
You commited the .gitignore. Also Honk.

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,6 @@
 *.rsc
 *.lk
 *.int
-*.backup
\ No newline at end of file
+*.backup
+config/admins.txt
```
Comment (Anonus) [inline .gitignore]: just commit it once and never have to do it again~

**Comment by AlexCrimson** [comment]
Dont forget the "clickbait" label.

File: .gitignore (hunk 1 above)
Comment (duncathan) [inline .gitignore]: Remove the .gitignore line and this will be s beautiful change

Comment (bgobandit) [inline .gitignore]: oranges said to add it goddamnit oranges

**Comment by Aranclanos** [comment]
when commiting make sure you select the files that you actually want to change

**Comment by bgobandit** [comment]
that didn't show up in there though

**Comment by Firecage** [comment]
Wait, I thought it was Honk Mother instead of Honkmother?....Was I wrong this entire time?

File: .gitignore (hunk 1 above)
Comment (optimumtact) [inline .gitignore]: no, I said to add it to you local repo, that has no relation to you adding it to the mainline

**Comment by Razharas** [comment]
I dont know what the fuck gitignore is and why are you touching it here so im not merging it until its there
@bgobandit

**Comment by optimumtact** [comment]
the file name literally tells you what it does, it lets you specify file mask matchers that git will quite literally ignore, changes to those files will not be committed unless you explicitly git add --force them (--force or something like that anyway)

**Comment by Razharas** [comment]
Do we really need adding admin txt into it as well as gitignore itself as i presume

**Comment by astralenigma** [comment]
I usually add admin.txt in the repo so that it isn't updated when I put my name to test stuff with admin powers.
Of course that file only gets used when the database is down when that happens all the coders will become admins on TG MUHAHAHAHA.

**Pull request closed without merging**